src/test/regress/expected/multi_mx_truncate_from_worker.out

```diff
@@ -76,6 +76,9 @@ INSERT INTO on_update_fkey_table SELECT i, i % 100  FROM generate_series(0, 1000
 -- now, show that TRUNCATE CASCADE works expected from the worker
 TRUNCATE "refer'ence_table" CASCADE;
 NOTICE:  truncate cascades to table "on_update_fkey_table"
+NOTICE:  truncate cascades to table "on_update_fkey_table_xxxxxxx"
```

Comment:
Why do you think this has changed?

Reply:
probably because there we have `citus.log_local_command = on` and we are logging the local execution of that TRUNCATE command, hence updated that test as well after introducing local TRUNCATE execution

Reply:
This file doesn't have any `log_local_command `?

Reply:
You're right, inspected the test there and we have both `citus.log_local_command = off` & `citus.log_remote_command = off`.

I think we normally do not log cascaded command execution when using remote connections however as we prefer there local execution from now on, those cascade logs are prompted by PostgreSQL's process utility.